planetiler-custommap/src/main/java/com/onthegomap/planetiler/custommap/validator/SchemaSpecification.java

```diff
@@ -55,6 +55,7 @@ public record OutputFeature(
     GeometryType geometry,
     @JsonProperty("min_zoom") Integer minZoom,
     @JsonProperty("max_zoom") Integer maxZoom,
+    @JsonProperty("min_pixel_size") Double minPixelSize,
```

Comment:
can you also add a test to `SchemaValidatorTest` to make sure this is being compared in the tests?